index.html

```diff
@@ -229,17 +237,74 @@ <h3>Multi-Post Stories</h3>
                                     <li><a href=""></a>Ruby</li>
                                 </ul>
                             </div>
-    
+
                         </div>
                         <div class="project-btn">
                             <button type="button" class="btn">see project</button>
                         </div>
                     </div>
-                    
+
+                </div>
+
+            </section>
+
+            <section class="work-container">
+                <div class="contact-us">
+                    <h3>CONTACT US</h3>
+                    <p>If you have an application you are interested in developing,
+                        a feature that you need built or a project that needs coding.
+                        I’d love to help with it </p>
+                </div>
+                <div class="form-sect">
+                    <form action="https://formspree.io/f/xleazodp" method="post">
+                        <ul>
+                            <li>
+                                <label for="name"></label>
+                                <input type="text" id="name" name="user_name" class="name-section" maxlength="30"
+                                    placeholder="Text input" required>
```

Comment:
Please write the correct placeholder for the name field. I may be `first name` or `full name`.